README.md

```diff
@@ -169,9 +169,20 @@ helm uninstall -n multicluster-controlplane multicluster-controlplane
 
 You can use clusteradm to access and join a cluster.
 
+1. Get the join token from controlplane:
 ```bash
 clusteradm --kubeconfig=<controlplane kubeconfig file> get token --use-bootstrap-token
-clusteradm join --hub-token <controlplane token> --hub-apiserver <controlplane apiserver> --cluster-name <cluster name>
+```
+
+2. Join a cluster using controlplane agent is now available in clusteradm(see this [PR](https://github.com/open-cluster-management-io/clusteradm/pull/338) for more details, you should build the latest code).
```

Comment:
Does it help on? `curl -L https://raw.githubusercontent.com/open-cluster-management-io/clusteradm/main/install.sh | bash` I guessed it should be. But `--singleton` in unknown flag. Do you know why?

Reply:
@clyang82 this script only downloads a tagged version of clusteradm https://github.com/open-cluster-management-io/clusteradm/blob/main/install.sh#L86, but the pr is still not included in any tagged version